index.html

```diff
@@ -181,7 +181,50 @@ <h3>Skills</h3>
                 <img id="narrow3" src="./images/light.png" alt="ico_light">
             </div>
         </div>
-    </section>   
+    </section>
+    <!--Forms section-->
+    <section>
+        <div class="ux">
+            <img id="uxlogo" src="./images/image_geometry_2.png" alt="logo">
+            <h2>Get stared</h2>
+            <p class="phrase">Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Voluptatum commodi aliquid quis pariatur totam explicabo
+                cupiditate, quaerat saepe dignissimos tempore nemo fugit,
+                porro illum aspernatur exercitationem rem ipsa voluptate quia.</p>
+            <form action="https://formspree.io/f/xayvddpz" method="post">
+                <ul class="data">
+                    <li class="liname">
+                        <label class="namelabel" for="name">Your name</label>
+                        <input class="name" type="text" id="name" name="user-name" placeholder="Francisco" maxlength="30" required>
+                    </li>
+                    <li class="liemail">
+                        <label class="emaillabel" for="email">Your email</label>
+                        <input class="email" type="text" id="email" name="user-email" placeholder="example@mail.com" required>
+                    </li>
+                    <li class="limsg">
+                        <textarea id="msg" name="user-message" maxlength="500" required>Enter your message here...</textarea>
+                    </li>
+                    <li>
+                        <a class ="txtbutton">Start collaboration</a>
+                    </li>
+                </ul>
+            </form>
+            <footer>
+                <div class="pjbutton1">
+                    <a class="actbutton2">Get my Resume</a>
+                    <img class="narrow1" src="./icons/ic_link_teal.png" alt="ico_download">
+                    <ul class="icons">
+                        <li><a><img src="/icons/ic_stackoverflow.png" alt=stackoverflow></a></li>
+                        <li><a><img src="/icons/ic_github.png" alt=github></a></li>
+                        <li><a><img src="/icons/ic_twitter.png" alt=twitter></a></li>
+                        <li><a><img src="/icons/ic_linkedin.png" alt=linkedin></a></li>
+                        <li><a><img src="/icons/ic_medium.png" alt=medium></a></li>
+                        <li><a><img src="/icons/ic_angellist.png" alt=angellist></a></li>
+                    </ul>
+                </div>   
+            </footer>    
```

Comment:
- [ ] Please add `.` before all images paths to make them work online because now all these images don't show up.
for example: ` <li><a><img src="./icons/ic_github.png" alt=github></a></li>`

- [ ] Please add the right widths and margins for all sections of your website and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160960692-3b196f89-0aee-43f2-8708-39e789cb6892.png)  | ![image](https://user-images.githubusercontent.com/68796232/160960716-bc5bab9a-35f3-4883-b2ba-e54467e7e995.png) |